models/productDao.js

```diff
@@ -0,0 +1,17 @@
+const { database } = require('./dataSource');
+const { distanceFormulaForQuery } = require('./enum');
+
+const getProductDistanceByUser = async ( latitude, longitude, distance ) => {
+  return await database.query(`
+    SELECT count(*)
+    FROM (
+    SELECT ${distanceFormulaForQuery.formula}
+    FROM products ) AS DATA
+    WHERE DATA.distance <= ?;`,
+    [ distance ]
+  );
+}
```

Comment:
formula 쿼리 객체를 수정하면서 다소 수정할 부분이 몇군데 보입니다.

enum이 아닌 쿼리 자체를 반환하는 모듈로서 수정 후 해당 코드도 수정해주세요!

추가로, query문 포맷팅 수정도 해주면 좋을 것 같습니다!
```suggestion
// example
const { distanceFormulaForQuery } = require('./fomualQuery');

const getProductDistanceByUser = async ( latitude, longitude, distance ) => {
  return await database.query(`
    SELECT count(*)
    FROM (
      SELECT ${distanceFormulaForQuery(latitude, longitude)}
      FROM products
    ) AS data
    WHERE data.distance <= ?;`,
    [ distance ]
  );
}
```